modules/backup-and-restore/pages/cross-cluster-backup.adoc

```diff
@@ -7,6 +7,35 @@ This page walks you through the process of restoring a cluster from a backup tha
 Restoring is an off-line operation.
 Your cluster is offline during the restore process.
 
+NOTE: Since v3.9.3, it is no longer required to create a backup for a backups metadata or upload the metadata to a new cluster.
```

Comment:
v3.9.2

Reply:
Thank you. Changed this with latest commit.